vendor/knative.dev/eventing/test/rekt/resources/containersource/containersource.yaml

```diff
@@ -49,7 +49,7 @@ spec:
     spec:
       containers:
       - name: heartbeats
-        image: registry.ci.openshift.org/openshift/knative-v1.8:knative-eventing-test-heartbeats
+        image: ko://knative.dev/eventing/cmd/heartbeats
```

Comment:
that's a wrong reference here, we don't do the `ko://`  this should be proper in midstream....